### ISSUE TITLE
Fabio support for specifying paths to the upstream

### ISSUE THREAD

**Issue by danlsgiga** [fabiolb/fabio#679]
As far as I could see, trying to do the following in Fabio is not supported:

```
route add app1 app1.example.com/.well-known/ http://app1.service.consul:8000/.well-known/
```

It seems to me that Fabio does not support specifying paths in the upstream, so `http://app1.service.consul:8000/.well-known/` is not recognized as a valid destination. I looked through the docs for any specific examples for this scenario but couldn't find any.

Is there any way to overcome this? Seems to be a pretty basic feature for a proxy.

**Comment by leprechau** [comment]
Not sure what you are trying to accomplish there?  The source and destination path are the same so why do you need to alter/specify the destination?

**Comment by leprechau** [comment]
You are correct in your observation though.  Fabio will use the same request path that it receives when it relays to the destination.  The only exception to this is the path stripping feature.  Fabio does not currently offer any other path rewriting or manipulation.

**Comment by danlsgiga** [comment]
Sorry, bad copy and paste here... here's the correct example:

`route add app1 app1.example.com/.well-known/ http://app1.service.consul:8000/v1/oauth/.well-known/`
